config/v1/types_infrastructure.go

```diff
@@ -301,6 +301,12 @@ type AWSPlatformSpec struct {
 	// There must be only one ServiceEndpoint for a service.
 	// +optional
 	ServiceEndpoints []AWSServiceEndpoint `json:"serviceEndpoints,omitempty"`
+
+	// userTags is a list of additional tags to apply to AWS resources created for the cluster. Changes made to userTags
+	// will not be reflected in existing AWS resources. The userTags are only applied to an AWS resource when the
+	// resource is created.
```

Comment:
@alexander-demichev is this how it works upstream or do they reconcile as well? I think we will want to mimic their behaviour ideally so we can make the behaviour consistent across various openshift topologies

Reply:
This also feels like we're implicitly saying we shouldn't change existing resources, and we're stating we won't.  This might require special handling depending on the component in the future.